trpcultivate_phenotypes/src/Plugin/Validators/ValueInList.php

```diff
@@ -24,27 +19,21 @@
  */
 class ValueInList extends TripalCultivatePhenotypesValidatorBase implements ContainerFactoryPluginInterface {
 
-  /**
-   *   This validator requires the following validator traits:
-   *   - ColumnIndices: Gets an array of indices corresponding to the cells in
-   *       $row_values to act on.
-   *   - ValidValues: Gets an array of values that are allowed within the cell(s)
-   *       located at the indices in getIndices().
+  /*
+   * Validator Traits required by this validator.
+   *
+   * - ColumnIndices: Gets an array of indices corresponding to the cells in
+   *   $row_values to validate.
+   * - ValidValues: Gets an array of values that are allowed within the cell(s)
+   *   located at the indices provided by getIndices().
    */
   use ColumnIndices;
   use ValidValues;
 
-  /**
-   * Constructor.
-   */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
-    parent::__construct($configuration, $plugin_id, $plugin_definition);
-  }
-
   /**
    * {@inheritdoc}
    */
-  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition){
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
Again, if we remove the constructor, shouldn't we also remove the create method?

Reply:
Addressed in commit [e4c13f8](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/e4c13f8ba2c3bbdcbd80802318948ffbc43929e4)!